cspell.json

```diff
@@ -1,5 +1,6 @@
 {
   "language": "en",
+  "dictionaries": ["softwareTerms"],
```

Comment:
If `softwareTerms` allows removing some words can you please do this in separate PR?

Reply:
Most of our custom words have been added to the `softwareTerms` dictionary in the mentioned PR. And changes to both `en_us` and `software_terms` at `cspell_dicts` will be merged in a single PR only.

Reply:
Also, just re-checked, all the dicts are auto included.